src/uniprotkb/components/entry/tabs/genomic-coordinates/GenomicCoordinates.tsx

```diff
@@ -0,0 +1,72 @@
+import { Loader, Message } from 'franklin-sites';
+
+import Coordinates from './Coordinates';
+import ContactLink from '../../../../../contact/components/ContactLink';
+
+import useDataApi from '../../../../../shared/hooks/useDataApi';
+
+import { proteinsApi } from '../../../../../shared/config/apiUrls';
+
+import { GenomicEntry } from './types';
+
+import tabsStyles from '../styles/tabs-styles.module.scss';
+
+type GenomicCoordinatesProps = {
+  primaryAccession: string;
+  title?: string;
+};
+
+const GenomicCoordinates = ({
+  primaryAccession,
+  title,
+}: GenomicCoordinatesProps) => {
+  const { loading, data, progress, error, status } = useDataApi<GenomicEntry>(
+    proteinsApi.coordinates(primaryAccession)
+  );
+
+  if (loading) {
+    return (
+      <div>
+        {title && <h3>{title}</h3>}
+        <Loader progress={progress} />
+      </div>
+    );
+  }
+
+  if (error && status !== 404) {
+    // TODO: use in-page error message
```

Comment:
Any blocker or could be done at this point?

Reply:
Done now